slee2/lab4_final.py

```diff
@@ -0,0 +1,57 @@
+__author__ = 'sLee'
+'''
+Lab3
+Samuel Lee
+Sept/28/2015
+Goal is to Write a Python module to verify that the Magic 1089 trick
+is indeed true.
+This program should:
+Have a function that takes a number as an argument and verifies that
+the digits are in decreasing order,
+returning True if they are and False if not.
+The digits must be strictly decreasing,
+since the Magic 1089 trick does not
+work for numbers like 988, where two digits are equal.
+# One way to do this is to convert the number to a string using
+the str() function, then use a for loop to compare neighboring characters.
+Have a function that takes a number as an argument. If the number is
+three digits, and the digits are in decreasing order, then the function
+should perform the above calculations and return the result
+(which should be 1089) as an integer.
+'''
+
+
+def magic_n(n):  # n is a list contain three int
```

Comment:
i just pushed it.

On Tue, Oct 13, 2015 at 9:03 PM, Kyle Tarplee <notifications@github.com>
wrote:

> In slee2/lab4_final.py
> <https://github.com/AndersonUniversity/CPSC4500test/pull/13#discussion_r41945791>
> :
>
> > +Have a function that takes a number as an argument and verifies that
> > +the digits are in decreasing order,
> > +returning True if they are and False if not.
> > +The digits must be strictly decreasing,
> > +since the Magic 1089 trick does not
> > +work for numbers like 988, where two digits are equal.
> > +# One way to do this is to convert the number to a string using
> > +the str() function, then use a for loop to compare neighboring characters.
> > +Have a function that takes a number as an argument. If the number is
> > +three digits, and the digits are in decreasing order, then the function
> > +should perform the above calculations and return the result
> > +(which should be 1089) as an integer.
> > +'''
> > +
> > +
> > +def magic_n(n):  # n is a list contain three int
>
> Put the description of the arguments (n in this case) in the docstring for
> the function. You don't want a comment here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AndersonUniversity/CPSC4500test/pull/13/files#r41945791>
> .
>